app/controllers/issues_controller.rb

```diff
@@ -101,6 +101,7 @@ def resolve_layout
   # Use callbacks to share common setup or constraints between actions.
   def set_issue
     @issue = Issue.find(params[:id] || params[:issue_id])
+    @club = @issue.club
```

Comment:
@orenmendelow whats this about?

Reply:
This is required for pulling theme colors into the sent issue page background. Here's what happens if I remove it:
![Screenshot 2024-10-29 at 9 01 02 PM](https://github.com/user-attachments/assets/d2f352a0-e54c-40ba-a5de-99aaee1fd53b)

Reply:
Ah I see. Ok